mobly/snippet/client_base.py

```diff
@@ -418,15 +418,9 @@ def handle_callback(self, callback_id, ret_value, rpc_func_name):
       The callback handler object.
     """
 
-  def stop_server(self):
-    """Proxy function of do_stop_server."""
-    self.log.debug('Stopping snippet %s.', self.package)
-    self.do_stop_server()
-    self.log.debug('Snippet %s stopped.', self.package)
-
   @abc.abstractmethod
-  def do_stop_server(self):
-    """Kills any running instance of the server."""
+  def stop(self):
```

Comment:
I'm not sure about changing this name, since it pairs with `start_server`?
If we want to change this, should we change `start_server` as well?
Would it be easier to just keep it as `stop_server`?

Reply:
ok, so this is really `clean_up`, which calls `stop_server`? lol
If we have a dedicated `start_server` as part of `initialize` in the interface, should we have a `stop_server` as part of `clean_up`?..
Basically, 有始有终？

Or is it not worth the trouble?

Reply:
There will be a `stop_server` in the subclass implementation. 

But I think we do not need to have it as an interface in base class. As we decided to make `clean_up` as an abstract method, it would be strange to have two abstract methods `clean_up` and `stop_server`, and their comments reminds that subclass must call `stop_server` in the implementation of `clean_up`.

We can just clarify that `clean_up` should clean all the acquired resources. Does that make sense?

Also, do you suggest to rename this function to `clean_up`?

Reply:
ok, let's follow the current plan